doc/HACKING/abbrevs-proposed.txt

```diff
@@ -0,0 +1,890 @@
+        intropoint
+
+isolate, isolation:
+        iso
+
+iterate, iterator:
+        iter
+
+key derivation function:
+        kdf
+
+key-and-value:
+        kv
+
+kilobytes:
+        kb
+
+length:
+        len
+
+less than:
+        lt
+
+library:
+        lib
+
+limit:
+        lim
+
+link specifier:
+        ls
+        lspec # prefer
+
+little-endian:
+        le
+
+managed proxy:
+        mp (testing only)
+
+manager:
+        mgr
+
+manipulate, manipulation:
+        manip
+
+maximal, maximum:
+        max
+
+mean time between failures:
+        mtbf
+
+memory:
+        mem
+
+message:
+        msg
+
+method:
+        ???
+
+microdescriptor:
+        md # prefer
+        microdesc # deprecate
+
+microseconds:
+        usec
+
+middle:
+        mid
+
+millisecond:
+        m
+        ms
+        msec # prefer
+
+minimal, minimum:
+        min
+
+miscellaneous:
+        misc
+
+monotonic:
+        mono
+
+multiplexer:
+        mux
+
+multiply:
+        mul # prefer
+        mult
+
+nanoseconds:
+        ns
+        nsec # prefer
+
+negate, negative:
+        neg
+
+network-order:
+        n
+
+networkstatus:
+        ns
+
+non-interruptable:
+        ni
+
+number:
+        n
+        num # prefer
+
+object:
+        obj
+
+onion router:
+        or
+
+operation:
+        ops
+
+option, optional:
+        opt
+
+or circuit:
+        ???
+
+or connection:
+        o
+        or_conn
+        orconn # prefer
+
+order-preserving encryption:
+        ope
+
+origin circuit:
+        ocirc # deprecate; ambiguous.
+
+out-of-bounds:
+        oob
+
+out-of-memory:
+        oom
+
+out-of-sockets:
+        oos
+
+output buffer:
+        outbuf
+
+padding:
+        pad
+
+parameter:
+        param
+
+password:
+        pass
+        passwd # preferred
+        pwd
+
+pinning:
+        pin
+
+pluggable transport:
+        pt
+
+pointer:
+        ptr
+
+position:
+        pos
+
+preferred:
+        pref
+
+primary:
+        prim
+
+privacy-enhanced mail:
+        pem
+
+private:
+        priv
+
+probability:
+        prob
+
+process:
+        ???
+
+process id:
+        pid
+
+protocol:
+        proto
+
+public, publish:
+        pub
+
+public key:
+        pk
+
+queue:
+        q
+
+random:
+        rand # preferred
+        rnd  # deprecate
+
+receive:
+        rcv  # deprecate
+        recv # preferred
+
+received:
+        rcdv
+
+receiver:
+        rcvr
+
+reference:
+        ref
+
+remove:
+        rm
+        rmv # preferred
+
+rendezvous:
+        rend
+
+rendezvous point:
+        rp
+
+reputation:
+        rep
+
+request:
+        req
+
+return:
+        ret
+
+round-trip time:
+        rtt
+
+router:
+        ??? # proposed: no abbreviation
+
+routerinfo:
+        ri
+
+routerstatus:
+        rs
+
+sandbox:
+        sb # only in sandbox code.
+
+second, secret, secure, security:
+        sec # too overloaded?
+
+secret key:
+        sk
+
+secret-to-key:
+        s2k
+
+serve, server:
+        serv
+
+service:
+        ??? # proposed: no abbrev?
```

Comment:
svc ?

Reply:
Yes, `svc` is commonplace, e.g., SUNRPC.